spec/chromium-spec.ts

```diff
@@ -2113,7 +2113,8 @@ describe('chromium features', () => {
       'chrome://gpu',
       'chrome://media-internals',
       'chrome://tracing',
-      'chrome://webrtc-internals'
+      'chrome://webrtc-internals',
+      'chrome://process-internals'
```

Comment:
`chrome.send` will not be available in this page due to [BINDINGS_POLICY_NONE](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/process_internals/process_internals_ui.cc;l=31). Maybe check for `window.hasOwnProperty('chrome') && window.location.host === "process-internals"`